rocketpy/Function.py

```diff
@@ -3034,6 +3035,27 @@ def __get__(self, instance, owner=None):
         return funcify_method_decorator
 
 
+def reset_funcified_methods(instance):
+    """Recalculates all the funcified methods of the instance. It does so by
+    deleting the current Functions, which will make the interperter redefine
+    them when they are called. This is useful when the instance has changed
+    and the methods need to be recalculated.
```

Comment:
This actually resets every cached_property, no? Not only funcified ones

Reply:
I don't think so. I believe the `__cached__` tag that was added only apply to `funcify`. They have no parallel in `cached_properties`.

Reply:
Okay, got it!